src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.js

```diff
@@ -95,8 +95,9 @@ class DatagridCell extends React.Component {
   }
 
   getStyle() {
-    const { width } = this.props;
-    return { width, minWidth: '2rem', maxWidth: width };
+    const { width, editable } = this.props;
+    if (editable) return { width, maxWidth: width };
+    return { width, minWidth: '4.625rem', maxWidth: width };
```

Comment:
actually this should be the opposite no ?

Reply:
then the columns will not display correctly

Reply:
minWidth: '4.625rem' was only for editable columns, for other ones we should keep the  '2rem